skyportal/plot.py

```diff
@@ -556,6 +545,45 @@ def add_axis_labels(plot, panel_name):
     plot.yaxis.axis_label = axis_label_dict[panel_name]['y']
 
 
+def check_visibility_on_phot_plot(user, show_all_filters, show_all_origins, label):
+    """Function to check if a photometry point should be displayed on the plot in accordance with the user's photometry plotting preferences.
```

Comment:
```suggestion
    """Function to check if a photometry point should be displayed 
    on the plot in accordance with the user's photometry plotting preferences.
```

Reply:
I would also break up the long lines in the Parameters block.

Reply:
fixed